etna/records/templatetags/records_tags.py

```diff
@@ -10,12 +11,24 @@
 
 
 @register.simple_tag
-def record_url(record: Union[Record, Dict[str, Any]]) -> str:
+def record_url(
+    record: Union[Record, Dict[str, Any]], is_editorial: bool = False
+) -> str:
     """
     Return the URL for the provided `record` dict; which could either be a
     full/transformed result from the fetch() endpoint, OR a raw result from
     the search() endpoint.
     """
+    if is_editorial and settings.FEATURE_RECORD_LINKS_GO_TO_DISCOVERY:
```

Comment:
Would it possible and better to have to only one condition - a flag value 'FEATURE_FLAG==<somevalue>`, and reduce dependency on is_editorial?

Reply:
Not really - We need `{% record_url %}` to work as normal in most places - except for where the records have been editorially added to content (e.g. on Insights pages)